README.md

```diff
@@ -1,7 +1,28 @@
-# FortiGate Autoscale Core module
-This is the Autoscale Core module written in TypeScript.
+# Autoscale Core
+The Core Library for the Fortinet Autoscale project.
 
-Note: This is a pre-release for internal development purposes.
+The Autoscale project provides a cloud-based multi-group auto scaling functionality for virtual machines of Fortinet products which have native HA features to form a cluster with failover protection.
+
+This project provides the fundamental, generic logics of a Hybrid Licensing architecture. It also includes implementations for certain popular cloud platforms.
+
+This project has the following features:
+ * Multi-group Hybrid Licensing models. (Hybrid, BYOL-Only, PAYG-Only)
+ * AWS Transit Gateway Integration (see: https://github.com/fortinet/fortigate-autoscale-aws)
+
+
+## Supported platforms
+This project supports autoscaling for the cloud platforms listed below:
+* Amazon AWS
+
+## NodeJS dependency
```

Comment:
This section heading does not make sense given the content below.

Reply:
> This section heading does not make sense given the content below.

Which line do you refer to? Mind giving the line number?

Reply:
The heading "NodeJS dependency" (line 17) doesn't make sense to me as NodeJS is only ever mentioned in the heading.

Reply:
NPM is a tool to manage NodeJS dependencies. that's why it has this heading.